DEVELOPMENT.md

```diff
@@ -46,6 +46,19 @@ To run a specific e2e test locally:
 $ make e2e-local TEST=TestCreateInstallPlanManualApproval
 ```
 
+##### Updating test images
```

Comment:
nit: should we also give the disclaimer that you need to be logged into quay from an account that has write access to the olmtest namespace?

Reply:
Done